SU-SWS/ace-sdssgryphon
# SDSS-816: Use serif font for block quote styles

#  READY FOR REVIEW

# Summary
- Changed the Block Quote GLOBALLY to a Serif
- Updated the Decanter to get the a11y changes. 

# Review By (Date)
- Soon

# Criticality
- How critical is this PR on a 1-10 scale? See [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to [localhost]/qa-typography-styles
4. Verify the block quotes are a serif font
5. Add a callout quote in sm, med, large to a news article.
6. Verify those also get the serif font. 
7. Decanter change should create no visible change. It is for the Screen readers. 

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?

## Backend / Functional Validation
### Code

### Code security

## General

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SDSS-816](https://stanfordits.atlassian.net/browse/SDSS-816)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)
